package.json

```diff
@@ -4,7 +4,8 @@
   "description": "The Apostrophe Content Management System.",
   "main": "index.js",
   "scripts": {
-    "test": "npm run lint && npm run audit && mocha",
+    "pretest": "npm run lint",
+    "test": "nyc --reporter=html mocha",
```

Comment:
It seems we're losing the npm audit step.

Reply:
`audit` was causing the test command to exit because it found vulnerabilities so tests weren't running. Could add audit back in, but might be better if audit is run as a separate step of the ci/cd since it's not really related to unit testing.